.semaphore/semaphore.yml

```diff
@@ -27,6 +27,33 @@ blocks:
               cache store m2 $HOME/.m2
             fi
       jobs:
+        - name: tests and deploy
+          commands:
+            - export ARGS="-DargLine='-Xms1024m -Xmx2048m'"
+            - mvn -e --no-transfer-progress clean integration-test failsafe:verify "${ARGS}"
+
+        - name: sevntu
+          commands:
+            - mvn -e --no-transfer-progress compile antrun:run@ant-phase-verify-sevntu -Psevntu
+
+        - name: codecov
+          commands:
+            - export USE_MAVEN_REPO="true"
+            - ./.ci/travis.sh jacoco
+
+        - name: codenarc analysis
+          commands:
+            - ./.ci/codenarc.sh
+
+        - name: no-exception-samples-ant
+          commands:
+            - ./.ci/no-exception-test.sh no-exception-samples-ant
```

Comment:
Should we just put these with other no-exceptions?

Reply:
Done

Reply:
Wait, not done

Reply:
Ok, done now